File: Java/Logic-2/closeFar.java
public boolean closeFar(int a, int b, int c) {
  int abDiff = Math.abs(a - b);
  int acDiff = Math.abs(a - c);
  int bcDiff = Math.abs(b - c);
  
  if (abDiff <= 1 && acDiff >= 2 && bcDiff >= 2) {
    return true;
    
  } else if (acDiff <= 1 && abDiff >= 2 && bcDiff >= 2) {
    return true;
    
  }
  
  return false;
}

File: Python/Logic-2/close_far.py
def close_far(a, b, c):
  
  a_b_diff = abs(a-b)
  a_c_diff = abs(a-c)
  b_c_diff = abs(b-c)
  
  if (a_b_diff <= 1 and a_c_diff >= 2 and b_c_diff >= 2
    or a_c_diff <= 1 and a_b_diff >= 2 and b_c_diff >= 2):
    return True
    
  return False
